lib/flop/operators.ex

```diff
@@ -254,9 +254,14 @@ defmodule Flop.Operators do
     end
   end
 
-  defp prelude(:split_search_text) do
+  defp prelude(:try_split_search_text) do
     quote do
-      var!(value) = Flop.Misc.split_search_text(var!(value))
+      var!(value) =
+        if is_bitstring(var!(value)) do
```

Comment:
`String.split` expects a binary, not a bitstring.

```suggestion
        if is_binary(var!(value)) do
```